### ISSUE TITLE
Issue Creating plot

### ISSUE THREAD

**Issue by JesseBonanno** [FuzzFoundation/WKTPlot#7]
Hi,

I have tried following along with the readme basic example. When i run the following code i get an error:
```
import shapely
from wktplot.wkt_plot import WKTPlot
plot = WKTPlot(title="California Counties", save_dir='/')
```

The error message is:
```
~\AppData\Local\Programs\Python\Python37\lib\site-packages\wktplot\wkt_plot.py in __init__(self, title, save_dir)
     38         output_file(save_dir / f"{title.lower().replace(' ', '_')}.html", title=title, mode="inline")
     39         self.figure = figure(title=title, x_axis_label="Longitude", y_axis_label="Latitude")
---> 40         self.figure.toolbar.autohide = True
     41 
     42     def add_shape(self, shape: ty.Union[str, BaseGeometry], **style_kwargs: dict):

~\AppData\Local\Programs\Python\Python37\lib\site-packages\bokeh\core\has_props.py in __setattr__(self, name, value)
    270 
    271             raise AttributeError("unexpected attribute '%s' to %s, %s attributes are %s" %
--> 272                 (name, self.__class__.__name__, text, nice_join(matches)))
    273 
    274     def __str__(self):

AttributeError: unexpected attribute 'autohide' to Toolbar, possible attributes are active_drag, active_inspect, active_scroll, active_tap, css_classes, disabled, height, js_event_callbacks, js_property_callbacks, logo, name, sizing_mode, subscribed_events, tags, tools or width
```

For reference i am using python version 3.7.3 and shapely version 1.8.0.

Also there is a minor mistake in the readme where save_dir should have an = after it in one segment of the code.

Best,
Jesse

**Comment by misimpso** [comment]
Interesting. Which version of `bokeh` are you using?

**Comment by JesseBonanno** [comment]
Looks like version 0.12.10

![image](https://user-images.githubusercontent.com/53243847/148360204-575a9ca5-3845-4f99-bcb1-76334ffa8580.png)

**Comment by misimpso** [comment]
That version of bokeh was released [over 4 years ago](https://github.com/bokeh/bokeh/releases/tag/0.12.10). I would suggest upgrading to the latest version of bokeh.

**Comment by JesseBonanno** [comment]
Okay. I was surprised when i installed the package that my version wasn't updated. I noticed a line was missing in setup.py and just sent a pull request to fix this issue.

**Comment by misimpso** [comment]
Oh, good find. Will check out the PR. Thanks!

**Comment by misimpso** [comment]
PR has been merged and new version of wktplot is out: 1.0.1
